### ISSUE TITLE
the "preprint-update" option

### ISSUE THREAD

**Issue by r-xue** [r-xue/ads2bibdesk#7]
We can add back the "preprint-update" option from the original **ads_bibdesk**, by scanning/updating article_bibcode in the arXiv style (something like *2019arXiv1234.5678*).

**Comment by caitlinmcasey** [comment]
Hi - I'm interested if you've gotten around to introducing this feature at all?  I've been doing it by hand but it has become a little overwhelming given the volume of JWST papers being posted on submission.  Also thanks in general for reviving ads2bibdesk -- really appreciated.

**Comment by r-xue** [comment]
No... life circumstances changed, and I didn't realize that I still had this open back since 19... 

will take a look, seems a straightforward thing to do if the API doesn't block us.
